File: Music.Client.Android/Java/short/app/src/main/java/com/zitherharpmusic/zhmshort/MainActivity.java
package com.zitherharpmusic.zhmshort;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.navigation.fragment.NavHostFragment;
import androidx.navigation.ui.NavigationUI;

import com.google.android.material.bottomnavigation.BottomNavigationView;
import com.zitherharpmusic.zhmshort.model.User;

public class MainActivity extends AppCompatActivity {
    private User user;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        getSupportActionBar().hide();
        BottomNavigationView navView = findViewById(R.id.nav_view);
        NavHostFragment navHostFragment = (NavHostFragment)getSupportFragmentManager().findFragmentById(R.id.nav_host_fragment);
        NavigationUI.setupWithNavController(navView, navHostFragment.getNavController());
    }

    public User getUser() {
        if (user == null) {
            user = new User(this);
        }
        return user;
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
        finish();
    }
}

File: Music.Client.Android/Kotlin/Music/short/src/main/java/com/zitherharp/music/shorts/ui/main/MainActivity.kt
package com.zitherharp.music.shorts.ui.main

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.navigation.fragment.NavHostFragment
import androidx.navigation.ui.AppBarConfiguration
import androidx.navigation.ui.setupActionBarWithNavController
import androidx.navigation.ui.setupWithNavController
import com.zitherharp.music.shorts.R
import com.zitherharp.music.shorts.databinding.ActivityMainBinding

class MainActivity: AppCompatActivity() {
    private val binding: ActivityMainBinding by lazy { ActivityMainBinding.inflate(layoutInflater) }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportActionBar?.hide()
        with(binding) {
            setContentView(root)
            val navController = mainFragment.getFragment<NavHostFragment>().navController
            val appBarConfiguration = AppBarConfiguration(setOf(
                R.id.navigation_home, R.id.navigation_explore,
                R.id.navigation_notification, R.id.navigation_user))
            navigationView.setupWithNavController(navController)
            setupActionBarWithNavController(navController, appBarConfiguration)
        }
    }
}